### PR TITLE
[TASK] Add more consistent example for ExtbasePluginEnhancer 

### PR DESCRIPTION
**Pull request by susannemoog**
The example was incomplete and inconsistent and has been adjusted to be more realistic (taken from the real news extension as inspiration).

Resolves: #966

### CODE REVIEW

**Comment by sypets** [comment]
> Resolves: #996
This is a PR. Wrong issue number?

**Comment by susannemoog** [comment]
> 
> 
> > Resolves: #996
> > This is a PR. Wrong issue number?

Those numbers start to look the same ;) 966 is correct.

File: Documentation/ApiOverview/Routing/AdvancedRoutingConfiguration.rst [hunk 1]
```diff
@@ -196,18 +196,50 @@ And generate the following URLs
...
+           requirements:
+             month: '\d+'
+             year: '\d+'
        defaultController: 'News::list'
        defaults:
          page: '0'
        requirements:
          page: '\d+'
+       aspects:
+         news-title:
```
Comment (mbrodala) [inline Documentation/ApiOverview/Routing/AdvancedRoutingConfiguration.rst:217]: This must be `news_title` instead AFAIS.

File: Documentation/ApiOverview/Routing/AdvancedRoutingConfiguration.rst [hunk 2]
```diff
@@ -196,18 +196,50 @@ And generate the following URLs
...
+           end: '100'
+         month:
+           type: StaticRangeMapper
+           start: '1'
+           end: '12'
+         year:
+           type: StaticRangeMapper
+           start: '1984'
+           end: '2525'
+         tag-name:
```
Comment (mbrodala) [inline Documentation/ApiOverview/Routing/AdvancedRoutingConfiguration.rst:233]: Same here: this should be `tag_name` instead.

**Review by mbrodala — CHANGES_REQUESTED** [review]

**Review by mbrodala — APPROVED** [review]

**Pull request merged**